commerce-theme-speedwell/commerce-theme-speedwell/src/css/_portlet.scss

```diff
@@ -121,13 +121,7 @@
   }
```

Comment:
looks like we already had indentation with spaces here.... it'd be nice to keep the files consistent as much as possible wrt indentation...